eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -77,7 +77,7 @@ stages:
                     steps:
                       - checkout: self
                       - script: |
-                          export DETECTED_PACKAGE_NAME=`ls $(Pipeline.Workspace)/${{parameters.ArtifactName}}/${{artifact.name}}-[0-9]*[0-9]*.[0-9]*.tgz`
+                          export DETECTED_PACKAGE_NAME=`ls $(Pipeline.Workspace)/${{parameters.ArtifactName}}/${{artifact.name}}/${{artifact.name}}-[0-9]*[0-9]*.[0-9]*.tgz`
```

Comment:
Part of the reason we are scoping the artifacts is to avoid needing to do these duplicated long greps in various places. Shouldn't we now be able to process all the files in the artifact.name directory?

Reply:
Definitely. I should have removed long grep and just have tgz. I will make this change.